pkg/services/ngalert/api/lotex_am.go

```diff
@@ -0,0 +1,170 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	apimodels "github.com/grafana/alerting-api/pkg/api"
+	"github.com/grafana/grafana/pkg/api/response"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	amSilencesPath    = "/api/v2/silences"
+	amSilencePath     = "/api/v2/silence/%s"
+	amAlertGroupsPath = "/api/v2/alerts/groups"
+	amAlertsPath      = "/api/v2/alerts"
```

Comment:
I'm not sure that these are the correct paths to forward to.

Reply:
It looks like Cortex doesn't yet support the v2 routes _at all_, which makes me think we should start with `v1` routes unless we need v2 functionality:

https://github.com/cortexproject/cortex/blob/master/pkg/api/api.go#L178-L183

cc @domasx2 @gotjosh @codesome Can we get by with just using v1?

Also, where are silences exposed in v1?

Reply:
It does support all of the am rotues, which we do use in cloud alerting ui plugin :)
I think the cortex server route you're looking for is here:
https://github.com/cortexproject/cortex/blob/master/pkg/api/api.go#L191

The prvious linked lines are for the AM config CRUD endpoints, which are implemented by cortex rather than alertmanager. The confusing am config endpoint name strikes again :D

Reply:
+1 to what @domasx2 said